### PR TITLE
Install helm, envsubst and gcloud pub/sub

### PR DESCRIPTION
**Pull request by KhadijaAmmar**
- Install helm
- Install gettext ( for envsubst)
- Install gcloud component pub/sub

### CODE REVIEW

**Review by fabong — APPROVED** [review]

**Comment by fabong** [comment]
@ccoeurderoy Can we also update the lts version?

**Comment by fabong** [comment]
Otherwise, LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM node:8.12.0
```
Comment (ccoeurderoy) [inline Dockerfile]: Could you take the latest version of node (which is called `carbon`)?

**Review by ccoeurderoy — CHANGES_REQUESTED** [review]

**Review by ccoeurderoy — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (KhadijaAmmar) [reply Dockerfile]: Done!

**Pull request merged**